Dnn.AdminExperience/ClientSide/SiteGroups.Web/src/components/Row.jsx

```diff
@@ -205,7 +205,6 @@ export default class SiteGroupRow extends React.Component {
         </div>
         <Collapse
           isOpened={this.props.isOpened}
-          style={{ width: "100%", height: "auto" }}
```

Comment:
this is internatlly managed by React-Collapse no need of it, explicitly providing it causeing overflow: hidden in container of React-Collapse on runtime